bin/disco_purge_snapshots.py

```diff
@@ -40,6 +41,8 @@
 from disco_aws_automation.disco_logging import configure_logging
 
 OLD_IMAGE_DAYS = 100
+DEFAULT_KEEP_LAST = 5
+NOW = datetime.now(pytz.UTC)
```

Comment:
I tried that originally but its actually pretty hard to mock `datetime.now()` 

http://blog.xelnor.net/python-mocking-datetime/